replica/table.cc

```diff
@@ -628,6 +628,13 @@ compaction_group& table::compaction_group_for_token(dht::token token) const noex
     return ret;
 }
 
+boost::integer_range<size_t> table::compaction_groups_for_token_range(dht::token_range tr) const noexcept {
+    size_t start = tr.start() ? compaction_group_for_token(tr.start()->value()).group_id() : size_t(0);
+    size_t end = tr.end() ? compaction_group_for_token(tr.end()->value()).group_id() : (_compaction_groups.size() - 1);
+
+    return boost::irange<size_t>(start, end + 1);
```

Comment:
Let's move away from boost stuff, you can use std::ranges::iota_view or even std::vector\<size_t>

Reply:
right.